webboot/webboot.go

```diff
@@ -212,7 +212,8 @@ func main() {
 			log.Fatal(err)
 		}
 	}
-
+	//the Request function sets up a DHCP confifuration for all interfaces,
+	//such as eth0, which is an ethernet interface.
```

Comment:
now for each argument, go ahead and describe in your own words what it's for. I know that may seem dumb but bear with me here. 
e.g. 
*ifName is the name of the interface
[hint: that's wrong. Look carefully at the default value of the argument to see why]

Reply:
but how? And it's not for all interfaces. Why is the name so weird?
^[we].*

and how does that end up being for all interfaces? Not to be mean but I suspect there's a bit of guessing in your answer. Not surprising or bad if so; this is a SIMPLE thing that hides a COMPLEX answer.